src/components/EthVideo.tsx

```diff
@@ -1,21 +1,20 @@
-import React from "react"
 import { Box, useColorModeValue } from "@chakra-ui/react"
 
-import darkVideo from "../../public/ethereum-hero-dark.mp4"
-import lightVideo from "../../public/ethereum-hero-light.mp4"
+import darkVideo from "@/public/ethereum-hero-dark.mp4"
+import lightVideo from "@/public/ethereum-hero-light.mp4"
 
-export interface IProps {
+type EthVideoProps = {
   className?: string
   videoSrc?: string
 }
 
-const EthVideo: React.FC<IProps> = ({ className, videoSrc }) => {
+const EthVideo = ({ videoSrc }: EthVideoProps) => {
```

Comment:
seems this `videoSrc` prop is never used, should be removed

Reply:
Okay, happy to... this was a remnant from when this component was originally created to allow potential reuse, but it was never utilized.